### ISSUE TITLE
[Bug] Installation of chocolatey package `sdio` silently fails on Windows 10

### ISSUE THREAD

**Issue by WetHat** [tunisiano187/Chocolatey-packages#2917]
Installation of the chocolatey package 'sdio' prints "Operating system not supported" but returns without further error. 

**To Reproduce**
1. Open a Powershell command prompt on Windows 10
2. Execute the command `PS> choco install sdio -y`
3. See "Operating system not supported" being printed to the console
4. Check that neither the shortcuts nor the executables are being available.

**Expected behavior**
The package should just install fine on Windows 10.

**Desktop (please complete the following information):**
 - OS:  Windows 10
 - Version 10.0.19041

**Error Analysis**

The installation script `chocolateyInstall.ps1` incorrectly compares strings and numbers when checking the OS version.

~~~ PowerShell
$os=(Get-WMIObject win32_operatingsystem).Version.split('.')
if(($os[0] -le 6) -and ($os[1] -le 3)) {
    Write-output "Operating system not supported"
    exit 0;
}
~~~

on Windows 10 the comparison attempted is:

~~~Powershell
PS> "10" -le 6
True
~~~

because the number 6 is _promoted_ to a string.

The correct comparison looks like:

~~~ Powershell
$os=(Get-WMIObject win32_operatingsystem).Version.split('.')
if(([int]$os[0] -le 6) -and ([int]$os[1] -le 3)) {
    Write-output "Operating system not supported"
    exit 1;
}
~~~

Please note that returning exit code 0 on error is not a good idea because it makes chocolatey think everything is ok.

**Comment by tunisiano187** [comment]
Correctes on this gît i though i already did it.
For the return 0, i dont l'une it neighter, but it's the way to get the automated testing passed.

**Comment by jbarthelmes** [comment]
@tunisiano187 See also #2912 #2913. GitHub says you merged both PRs but only the second (#2913) is in master.

**Comment by tunisiano187** [comment]
Thanks been changed but not in thé latest version on chocolatey, it will be in the next release, sorry
